src/sentry/receivers/outbox/region.py

```diff
@@ -41,7 +35,7 @@ def process_audit_log_event(payload: Any, **kwds: Any):
         log_rpc_service.record_audit_log(event=AuditLogEvent(**payload))
 
 
-@receiver(process_region_outbox, sender=OutboxCategory.ORGAUTHTOKEN_UPDATE)
+@receiver(process_region_outbox, sender=OutboxCategory.UPDATE_ORGAUTHTOKEN_USED)
```

Comment:
Is `UPDATE_ORGAUTHTOKEN_USED` a permanent name? It doesn't follow the naming conventions that other outbox categories to of `thing_action`.

Reply:
Good call, I can rename easily.